drivers/volume/csi/csi.go

```diff
@@ -975,13 +1235,42 @@ func (c *csi) getCSIBackupObject(backupName, backupNamespace string) (*csiBackup
 		return nil, err
 	}
 
-	cbo := &csiBackupObject{}
-	err = json.Unmarshal(backupObjectBytes, cbo)
+	cboCommon := &csiBackupObject{}
+	if c.v1SnapshotRequired {
+		type CsiBackupObjectv1 struct {
```

Comment:
As mentioned in the px-backup PR, let's add these to `csicommon`

Reply:
stork doesn't define csiCommon package neither import it, it is defined in px-backup, And possibly we do not wanted create dependency of stork on px-backup. Not sure of the history behind csiCommon though.

I have shifted the definition of structs  to the place where in csiBackupObject{} is defined to make the code looks cleaner. Please let me know ur thoughts on this.

Reply:
Sounds good to me.